### PR TITLE
Updated release script

### PR DESCRIPTION
**Pull request by davidvarga93**
refs: MBL-15741
affects: Student, Teacher, Parent
release note: none

test plan:
Download the modified release.config from 1password in order to execute the script

### CODE REVIEW

**Comment by inst-danger** [comment]
<details><summary>Parent Build QR Code:</summary>

</details>

**Comment by inst-danger** [comment]
<details><summary>Teacher Build QR Code:</summary>

</details>

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 91.65% | 91.66% | -0%

[MBL-15741](https://instructure.atlassian.net/browse/MBL-15741)

#### Affected Apps: Student, Teacher, Parent
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by inst-danger** [comment]
<details><summary>Student Build QR Code:</summary>

</details>

File: scripts/build_automation/release.sh [hunk 1]
```diff
@@ -89,9 +76,14 @@ curl https://app.bitrise.io/app/$SLUG/build/start.json --data '{
                 "is_expand": true,
                 "mapped_to": "APP_RELEASE_VERSION",
                 "value": "'"$2"'"
-            }
+            },
+			{
```
Comment (vargaat) [inline scripts/build_automation/release.sh:80]: Indentation seems to be off.

**Review by vargaat — APPROVED** [review]
QA+1, I was able to trigger the bitrise release workflow.

**Pull request merged**